sidedata.go

```diff
@@ -0,0 +1,24 @@
+package ffprobe
+
+import "errors"
+
+const sideDataTypeField = "side_data_type"
+
+// ErrSideDataNotFound is a sentinel error used when a queried side data does not exist
+var ErrSideDataNotFound = errors.New("side data not found")
+
+// SideDataList is a json data structure to represent stream side data
+type SideDataList []Tags
+
+func (s *SideDataList) GetSideData(sideDataType string) (Tags, error) {
```

Comment:
Can we get a documentation comment on this (public) function?

Reply:
Oh yep sorry 👍